### PR TITLE
Add useful logs on funds migration methods

### PR DESCRIPTION
**Pull request by nathanieliov**
Add useful logs on funds migration methods

### CODE REVIEW

**Review by alexjavabraz — APPROVED** [review]
SGTM

File: rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java [hunk 1]
```diff
@@ -1168,6 +1184,7 @@ private void processPegoutsInBatch(
 
                 // Mark UTXOs as spent
                 List<UTXO> selectedUTXOs = result.getSelectedUTXOs();
+                logger.debug("[processPegoutsInBatch] Use {} UTXOs for this pegout", selectedUTXOs.size());
```
Comment (kelvinator07) [inline rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java:1187]:
```suggestion
                logger.debug("[processPegoutsInBatch] used {} UTXOs for this pegout", selectedUTXOs.size());
```

File: rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java [hunk 2]
```diff
@@ -996,13 +1001,24 @@ private void migrateFunds(
         BtcTransaction btcTx = createResult.getLeft();
         List<UTXO> selectedUTXOs = createResult.getRight();
 
+        logger.debug(
+            "[migrateFunds] consumed UTXOs: {}",
```
Comment (kelvinator07) [inline rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java:1005]:
```suggestion
            "[migrateFunds] consumed {} UTXOs",
```

File: rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java (hunk 1 above)
Comment (nathanieliov) [reply rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java]: Done.

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**